src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java

```diff
@@ -658,7 +659,9 @@ protected RepositoryData callCreateRepoApi(String projectKey, Repository repo)
                   + " - no response from endpoint, please check logs",
               repo.getName(), projectKey));
     }
-    setRepositoryAdminPermissions(data, projectKey, ID_GROUPS, repo.getUserGroup());
+    if (project.isSpecialPermissionSet()) {
```

Comment:
after discussing this we introduce following changes:
if special permission was set
- admin permission to the repo admin group
- admin permission to the repo admin user
if special permission was not set
- admin permission to the repository creator
in any case we will set
- write permission to repo user group

Reply:
After a deeper review, we find:

- If special permissions are set, permissions are correctly applied at project level and inherited at repo level.
- If special permissions are not set, nobody gets admin permissions at project level and the default user group gets write permissions, as desired. Providing admin permissions to the repository creator does not seem possible. It may even be a technical user.

So, everything is fine when special permissions are set.
I will create a table with the currently applied permissions for further discussion about the desired behaviour when special permissions are not set.

Reply:
This conversation is around outdated code. Resolving the conversation.